src/main/app/dashboard/idea/idea.component.html

```diff
@@ -0,0 +1,65 @@
+<div class="idea-card mb-4 active">
```

Comment:
Assuming this will be updated in next sprint.

Reply:
yes, this is just a placeholder for now, it will be changed in next sprint.